site/docs/v1/tech/apis/system.adoc

```diff
@@ -354,4 +356,59 @@ The specific contents of the JSON body are not documented here. If you choose to
 |===
 
 
+[NOTE]
+====
+This API has been available since 1.26.0
+====
+
+== Retrieve System Metrics Using Prometheus
+
+This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to setup Prometheus with the FusionAuth metrics endpoint.
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/prometheus/metrics#
+--
+
+=== Authentication
+This endpoint requires `basic auth` authentication method. Please refer to the link:/docs/v1/tech/apis/authentication#basic-authentication-with-api-key[Authentication] document for more details about `basic auth` authentication method using API keys.
+
+=== Request Parameters
+There are no request parameters required with this API.
+
+
+=== Response
+The response to this API call is various metrics that are currently available. The available metrics in this response is subject to change. The following types of gauges are currently available.
+
+* JVM Buffers - A set of gauges for the count, usage, and capacity of the JVM's direct and mapped buffer pools.
```

Comment:
should these be broken out to an include file shared between here and the https://github.com/FusionAuth/fusionauth-site/blob/master/site/docs/v1/tech/tutorials/prometheus.adoc file?